Dockerfile

```
@@ -0,0 +1,21 @@
+FROM python:3.7-buster as base
+RUN curl -sSL https://install.python-poetry.org | python3 -
+ENV PATH "$PATH:/root/.local/bin/"
+
+RUN mkdir /app
+WORKDIR /app
+
+COPY poetry.lock  poetry.toml pyproject.toml ./
+RUN poetry install
+
+
+FROM base as production
+COPY ./todo_app app/todo_app

```

Currently this is copying to "app" inside the current working directory, i.e. /app/app/todo_app.
Destination needs to start with a slash to be absolute:

```suggestion
COPY ./todo_app /app/todo_app
```
Alternatively you could make it relative to the current folder with `COPY ./todo_app ./todo_app`
Then it will use WORKDIR and you don't need to worry about keeping this in sync with any other commands.